lib/round.rb

```diff
@@ -0,0 +1,111 @@
+
+class Round 
+
+    attr_reader :deck, :turns, :num_correct
+    def initialize(deck)
+        @deck = deck
+        @turns = []
+        @num_correct = []
+ 
+end
+
+def current_card
+    @deck.cards[@turns.count]
+end
+
+def take_turn(guess)
+    new_turn = Turn.new(guess, current_card)
+    @turns << new_turn
+    if new_turn.correct?
+        @num_correct << new_turn
+    end
+    new_turn
+end
+
+def number_correct
+ @num_correct.count    
+end
+
+
+def number_correct_by_category(category)
+    num_correct_in_category = []
+@turns.each do |turn|
+    if turn.correct? && turn.card.category == category
+        num_correct_in_category << turn
+    end
+end
+
+num_correct_in_category.count
+
+end
+
+# @turns.select { |turn| turn.card.category ==  info }.count.to_f
+
+def total_number_of_cards_in_category(category)
+    number_of_cards_in_category = []
+    turns.select do |turn|
+       if turn.card.category == category
+        number_of_cards_in_category << turn
+        end
+    end
+    number_of_cards_in_category.count
+end
```

Comment:
You're using `select` AND a placeholder, which kind of makes the use of `select` unnecessary. We use `select` to return a subset of the original array. To still use `select`, you could do something like: 
```ruby 
    cards_in_category = turns.select do |turn|
               turn.card.category == category
     end.count
```